root/etc/e-smith/templates/etc/httpd/admin-conf/httpd.conf/32tls_policy_20180330

```diff
@@ -0,0 +1,6 @@
+{
+ my $tlsPolicy = $tls{'policy'} || '0';
+ if ($tlsPolicy >= 20180330) {
+    $OUT .= "#Only TLS1.2 cipher\nSSLCipherSuite EDH+aRSA+AES256:EECDH+aRSA+AES256:!SSLv3\n";
```

Comment:
I'd prefer the multiline string syntax, like you did with httpd because \n is hard to read.

Reply:
ok done